src/rpm/headers/header.rs

```diff
@@ -383,16 +382,6 @@ impl Header<IndexSignatureTag> {
         Ok(())
     }
 
-    #[inline]
-    pub fn get_file_ima_signatures(&self) -> Result<&[String], RPMError> {
-        self.get_entry_data_as_string_array(IndexSignatureTag::RPMSIGTAG_FILESIGNATURES)
-    }
-
-    #[inline]
-    pub fn get_file_ima_signature_length(&self) -> Result<u32, RPMError> {
```

Comment:
Sidenote, this kinda feels like an internal detail, when exposed to the user Rust types ought to know their own lengths.  I haven't looked into it deeply though - does that sound right?

Reply:
Sounds good to me

Reply:
Are we 100% certain the signatures are not binary and should never contain a null character except as termination?

Reply:
https://github.com/rpm-software-management/rpm/blob/master/docs/manual/tags.md#optional-file-information

Claims they are hex-encoded, so I suppose it should be fine.

Reply:
Yeah, sure of that.